frontend/packages/console-dynamic-plugin-sdk/src/extensions/details-item.ts

```diff
@@ -0,0 +1,67 @@
+import { ExtensionK8sModel } from '../api/common-types';
+import { Extension, ExtensionDeclaration, CodeRef } from '../types';
+import { K8sResourceCommon } from './console-types';
+
+export type DetailsItemColumn = 'right' | 'left';
+
+export type DetailsItemComponentProps<R extends K8sResourceCommon = K8sResourceCommon, V = any> = {
+  /** The subject resource */
+  obj: R;
+
+  /** The path property provided by the extension */
+  path: string;
```

Comment:
Minor: It's only defined if the path value of the extension is defined.

```suggestion
  path?: string;
```